README.markdown

```diff
@@ -468,7 +437,7 @@ Prefer the shortcut versions of type declarations over the full generics syntax.
 **Preferred:**
 ```swift
 var deviceModels: [String]
-var employees: [Int: String]
+var employees: [Int : String]
```

Comment:
IMO the space before the colon, or lack thereof, is a visual cue:

`a: b` means "variable is a type"
`a : b` means "type maps to a type"